### DISCUSSION TITLE
lesson 3 fund me in binance

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#443] [Q&A] [answered]
Hello,

How to use **fund me** (FundMe.sol) with BEP20  token?

please answer

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @gunomnifluencer Binanace Smart Chain is a fork of Ethereum so it works as same as any other ERC20 token, so if you want the users to pay with that token you have to approve it and require payment with it, use this as example:
```SOlidity

 function pay(IERC20 Token) public payable {
        require(
            Token.transferFrom(
                msg.sender,
                address(this),
                amount
            )
        );
}
```
